nodebuilder/das/config.go

```diff
@@ -16,7 +18,9 @@ type Config das.Parameters
 //
 // TODO(@derrandz): Address #1261
 func DefaultConfig() Config {
-	return Config(das.DefaultParameters())
+	cfg := das.DefaultParameters()
+	cfg.SampleTimeout = modp2p.BlockTime * time.Duration(cfg.SampleTimeout)
```

Comment:
This seem to make the timeout huge because it take the default and then additionally multiples it